polling/assets.go

```diff
@@ -67,9 +69,10 @@ type PegItems struct {
 	When  string
 }
 
-func (p *PegItems) Clone() PegItems {
+func (p *PegItems) Clone(randomize float64) PegItems {
 	np := new(PegItems)
-	np.Value = p.Value
+	np.Value = p.Value + p.Value*(randomize/2*rand.Float64()) - p.Value*(randomize/2*rand.Float64())
```

Comment:
Without being seeded first the function `rand.Float64()` will always return the same value for everyone using the default seed of 1

Reply:
something like `rand.Seed(rand.Int63())` at startup is all that's needed

Reply:
This is to provide random data for testing the OPR record grading.  We really don't care if it is really random.

Reply:
`2*rand.Float64` isnt really needed then and can simply be replaced with the hardcoded number `1.2` because that's what it will always return.